ui/static/ui/js/listing.js

```diff
@@ -1,6 +1,78 @@
 define(['jquery', 'bootstrap', 'icheck', 'csrf'], function(jQuery) {
   'use strict';
 
+  /* global $ */
+
+  function formatGroupName(string) {
+    string = string.charAt(0).toUpperCase() + string.slice(1);
+    return string.substring(0, string.length - 1);
+  }
+
+  /* This is going to grow up to check whether
+   * the name is a valid Kerberos account
+   */
+  function isMITEmail(kerberosId) {
+    var email = kerberosId + '@mit.edu';
```

Comment:
This ends up being really confusing in the UI:
![image](https://cloud.githubusercontent.com/assets/596029/8462489/4a0f6a4c-2002-11e5-8378-813a03e91f73.png)
when my e-mail address on the system is actually `x@1.com`  I think you should just show the username only.  That will be more flexible and doesn't require this to be hard coded in if anyone else wanted to use this.